tests/playbooks/tests_bond.yml

```diff
@@ -13,6 +13,19 @@
     - name: "INIT Prepare setup"
       debug:
         msg: "##################################################"
+    - name: Verify DNS and ping still work
+      shell: |
+        set -euxo pipefail
+        for host in mirrors.fedoraproject.org mirrors.centos.org; do
+          if ! getent hosts "$host"; then
+            exit 1
+          fi
+          if ! ping -c5 "$host"; then
+            exit 1
+          fi
+        done
+      when: ansible_facts["distribution"] == "CentOS"
+      changed_when: false
```

Comment:
If you intend to keep this code in, then it should be its own task file.

Reply:
I am not planning to keep this code in, but just for temporary debugging purposes.